api/src/organizational-entities/infrastructure/repositories/data-protection-officer.repository.js

```diff
@@ -35,6 +36,10 @@ async function create(dataProtectionOfficer) {
   return new DataProtectionOfficer(dataProtectionOfficerRow);
 }
 
+async function deleteDpoByOrganizationId(organizationId) {
+  await knex(DATA_PROTECTION_OFFICERS_TABLE_NAME).where({ organizationId }).delete();
+}
```

Comment:
Créer cette nouvelle fonction en l'équipant de base pour pouvoir être transactionnelle : 
```suggestion
async function deleteDpoByOrganizationId(organizationId) {
  const knexConn = DomainTransaction.getConnection();
  await knexConn(DATA_PROTECTION_OFFICERS_TABLE_NAME).where({ organizationId }).delete();
}
```